share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml

```diff
@@ -206,7 +206,7 @@ spack:
     #- umpire # unsatisfiable concretization conflict w/ blt
 
   - arch:
-    - '%gcc target=x86_64'
+    - '%gcc target=x86_64_v3'
```

Comment:
Does this offer any value over `packages:target:[x86_64_v3]` or vice versa? Are both needed?

Reply:
No, they are not both needed. The configuration:
```yaml
packages:
  all:
    target: [x86_64_v3]
```
is just a soft preference for concretization. What is used in a matrix instead is a hard setting, since it is used to construct a spec literal. If we specify targets as a hard setting the soft preference is not relevant.